Tests/Fixtures/StencilContexts/Plist/all.yaml

```diff
@@ -74,7 +74,15 @@ files:
               type: "String"
             Kits:
               element:
-                type: "Dictionary"
+                items:
+                - properties:
+                    KitInfo:
+                      properties: {}
+                      type: "Dictionary"
+                    KitName:
+                      type: "String"
+                  type: "Dictionary"
+                type: "Any"
```

Comment:
Isn't that a regression that we go from `type: Dictionary` to `type: Any`? I mean, Sure we also now have `items` and see that we have 2 items, one being a dict and the other being a string… so that's better… but still not sure why the parent element `"Kits` is reported as `Any` now and not `Dictionary`

I guess it depends what definition we give for the metadata we name `Dictionary`: is it supposed to mean "Swift's `Dictionary<Key, Value>`" or "ObjC's `NSDictionary`, / generic _dictionary_ concept"? If it's supposed to mean the Swift & generic type, then I get why it's reported as `Any` here because we have an heterogeneous dict… but then maybe we should add some notes in the SwiftGen Contexts docs then that `Dictionary` means "homogeneous / same key/value types" to just be extra clear about the meaning of the `type` for those metadata structures :wink: